presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -4,6 +4,9 @@ Release 0.273
 
 .. warning::
 
+   There is a regression that results in failures for queries referencing columns with
```

Comment:
How about `There is a regression causing query failures when referencing columns with`?